lib/json_api/view.ex

```diff
@@ -47,9 +47,8 @@ defmodule ElixirTools.JsonApi.View do
   @spec convert_key(String.t() | atom) :: String.t()
   defp convert_key(key), do: ensure_dash(to_string(key))
 
-  @spec ensure_dash(any) :: any
-  defp ensure_dash(value) when is_binary(value), do: String.replace(value, "_", "-")
-  defp ensure_dash(value), do: value
+  @spec ensure_dash(String.t()) :: String.t()
+  defp ensure_dash(value), do: String.replace(value, "_", "-")
```

Comment:
should this be in this PR?

Reply:
yep, specs have been broken with elixir 1.9.1

Reply:
removing the guard clause will result in errors in the JSON api views. What is the exact error you're getting?

Reply:
https://circleci.com/gh/digitalorigin/pg-elixir-tools/252 and it's logical because we call `ensure_dash` only from `convert_key` and do `to_string` beforehand - so it's can be only String on input

Reply:
I think it's fine, I double checked and it seems like you are right. Just test it before updating the version!